lessons/students/marking_period_1/week_2/10_functions_and_scope.py

```diff
@@ -171,3 +171,6 @@ def func3():
 func3()
 
 print(s3)
+
+# TODO: Section 1 of TODO 10
+####################################################################################################
```

Comment:
If this is the end of the lesson, you don't need the 100 #s. You can delete line 176, but then hit enter again so there is a blank line at the end of the file (line 176). Thats why there is a red error mark there.